src/orion/core/io/space_builder.py

```diff
@@ -218,25 +232,42 @@ def build(self, name, expression):
 class SpaceBuilder(object, metaclass=SingletonType):
     """Build a `Space` object form user's configuration."""
 
-    USERCONFIG_OPTION = '--config='
+    # TODO Expose these 4 USER oriented goodfellows to a orion configuration file :)
     USERCONFIG_KEYWORD = 'orion~'
-    USERARGS_SEARCH = r'\W*([a-zA-Z0-9_-]+)~(.*)'
     USERARGS_TMPL = r'(.*)~(.*)'
+    USERARGS_NAMESPACE = r'\W*([a-zA-Z0-9_-]+)'
+    USERARGS_CONFIG = '--config='
+
+    EXPOSED_PROPERTIES = ['trial.hash_name', 'trial.full_name', 'exp.name']
```

Comment:
If I understand properly, if there is some `$trial.hash_name` in the command line of the user, this will be replaced by the actual trial.hash_name provided by orion. Is that right?

Reply:
Nevermind